src/components/wallet/Wallet.js

```diff
@@ -306,26 +306,26 @@ export function Wallet() {
                 <div className='left'>
                     <div className='tab-selector'>
                         <div
-                            className={classNames(['tab-balances', tokenView !== 'fungibleTokens' ? 'inactive' : ''])}
-                            onClick={() => setTokenView('fungibleTokens')}
+                            className={classNames(['tab-balances', tab ? 'inactive' : ''])}
+                            onClick={() => setTab('')}
                         >
                             <Translate id='wallet.balances' />
                         </div>
                         <div
-                            className={classNames(['tab-collectibles', tokenView !== 'nonFungibleTokens' ? 'inactive' : ''])}
-                            onClick={() => setTokenView('nonFungibleTokens')}
+                            className={classNames(['tab-collectibles', tab !== 'collectibles' ? 'inactive' : ''])}
+                            onClick={() => setTab('collectibles')}
                         >
                             <Translate id='wallet.collectibles' />
                         </div>
                     </div>
-                    {tokenView === 'fungibleTokens' &&
+                    {!tab &&
```

Comment:
To guard against an invalid tab param, e.g. `tab=blabla` that would result in neither balances nor collectibles being rendered, we can do:

```
{tab === 'collectibles' ?
    <NFTs tokens={sortedNFTs} />
    :
    <FungibleTokens
        balance={balance}
        tokensLoader={tokensLoader}
        sortedTokens={sortedTokens}
    />
}
```

And then just change `L309` to:
`className={classNames(['tab-balances', tab === 'collectibles' ? 'inactive' : ''])}`

Reply:
Thanks Patrick! good catch. fixed now.